modules/govuk_elasticsearch/templates/es-prune-snapshots.erb

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+set -e
+
+# Redirect stdout and stderr to syslog
+exec 1> >(/usr/bin/logger -s -t $(basename $0)) 2>&1
+
+ES_REPO=<%= @es_repo %>
+LIMIT=<%= @es_snapshot_limit %>
+
+# Get a list of snapshots in the specified repo
+
+SNAPSHOTS=`/usr/bin/curl -s -XGET "127.0.0.1:9200/_snapshot/"${ES_REPO}"/_all" | /usr/bin/jq -r ".snapshots[:-${LIMIT}][].snapshot"`
```

Comment:
After a quick parse It seems version 1.4.x doesn't support cat snapshots.  I'll look a little more just in case it's referenced differently.